src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,91 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationServiceImpl regService = new RegistrationServiceImpl();
+    private User user = new User();
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("Ivan_Litvynets");
+        user.setPassword("qwerty123456");
+        user.setAge(23);
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_User_Ok() {
+        assertEquals(user, regService.register(user));
+        assertTrue(Storage.people.contains(user));
```

Comment:
U can check only one condition. It's enough to check if method returns user after registration